### DISCUSSION TITLE
Cloned updated version of 'development' branch to repository.

### DISCUSSION THREAD

**Discussion by MarkW06071969** [MarkW06071969/Tafe202206#6] [Announcements]
Hi all,

I have cloned the updated 'development' branch to the repository' with the following modifications:

Horizontally centered all components on the 'Main Menu' interface.
Added code in the 'Maths Calculator' button click event handler to run the 'Maths Calculator' application.
Added code in the 'Exit' button click event handler to exit the application.
Added stubs (commented out code) in all of the other button click event handlers to refer to further development of their
respective applications.

Stephen has helpfully suggested the names of the applications and assigned whom will be doing which. I am happy with that.

Thanks everyone.

**Comment by StephenHillenbrand** [comment]
Thanks Mark, great stuff

**Comment by StephenHillenbrand** [comment]
I'll check out the development branch today and complete the Currency Calculator and get it up by tonight. I haven't done any C# GUI coding in WPF but i think the underlaying layouts are the same and should be fine. I'm thinking i will just place everything in a Grid Layout  as the Currency Calculator mockup screens seem to be in three column fashion.

Does anyone see any problems using GRID layout?
